tsl/test/sql/bgw_custom.sql

```diff
@@ -241,5 +247,27 @@ SELECT wait_for_job_to_run(:job_id_4, 1);
 -- Chunk compress stats
 SELECT * FROM _timescaledb_internal.compressed_chunk_stats ORDER BY chunk_name;
 
+-- Decompress chunks
+SELECT decompress_chunk(c) FROM show_chunks('conditions') c;
+
+-- Chunk compress stats
```

Comment:
I don't think we really need this output since the test is really about the procedure.

Reply:
Fixed